src/pymovements/dataset/dataset.py

```diff
@@ -763,6 +765,12 @@ def compute_event_properties(
             if column != 'filepath'
         ]
 
+        existing_columns = set(self.events[0].columns) & set(event_properties)
```

Comment:
`event_properties` can be passed either as a list of `str` or as a dictionary, that's why some of the tests fail:

```python
       existing_columns = set(self.events[0].columns) & set(event_properties)
       TypeError: unhashable type: 'dict'
```

Just include these lines before and then use `set(event_property_names)`:

```python
if isinstance(event_properties, dict):
   event_property_names = event_properties
elif isinstance(event_properties, dict):
   event_property_names = list(event_properties.keys())
else:
   raise TypeError(f'event_properties must be of type str or dict, but is {type(event_properties)}')
```